model/rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  def initialize(person, book)
+    @book = book
+    book.rentals << self
```

Comment:
- [ ] In accordance to the requirement, kindly implement `@date` instance variable and it should be initialized in the constructor. Note: Don't forget to implement setter and getter for **@date** (`attr_accessor`).